salesforce/bulkwrite.go

```diff
@@ -52,10 +52,15 @@ type BulkWriteParams struct {
 	Mode BulkWriteMode
 }
 
+type BulkWriteParams BulkJobParams
```

Comment:
bulk write endpoint and bulk delete endpoints are the same, so I renamed the type struct. But for backward compatibility, we keep `BulkWriteParams` to not break existing usages.

Reply:
You can keep it as BulkWrite, delete is technically a form of write. `BulkJob` is too ambiguous.